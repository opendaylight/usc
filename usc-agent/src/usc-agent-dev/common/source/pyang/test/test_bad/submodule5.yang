submodule submodule5 {

  belongs-to composite {
    prefix c; 
  }

  include submodule4;

  augment "/c:new" {  // error; defined in non-included submodule2
    uses c:g1; // error; defined in non-included submodule2
  }
}
