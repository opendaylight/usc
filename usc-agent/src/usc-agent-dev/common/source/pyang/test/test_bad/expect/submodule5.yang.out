submodule5.yang:9: error: NODE_NOT_FOUND
submodule5.yang:10: error: GROUPING_NOT_FOUND
submodule2.yang:7: error: BAD_IMPORT
submodule2.yang:7: warning: UNUSED_IMPORT
